### PR TITLE
wgengine/netstack: in netstack/hybrid mode, fake ICMP using ping command

### PR DESCRIPTION
**Pull request by bradfitz**

### CODE REVIEW

File: wgengine/netstack/netstack.go [hunk 1]
```diff
@@ -377,11 +380,48 @@ func (ns *Impl) shouldProcessInbound(p *packet.Parsed, t *tstun.Wrapper) bool {
...
+	if !userPingSem.TryAcquire() {
+		return
+	}
+	defer userPingSem.Release()
+
+	t0 := time.Now()
+	var err error
+	switch runtime.GOOS {
+	case "windows":
+		err = exec.Command("ping", "-n", "1", "-w", "3000").Run()
```
Comment (crawshaw) [inline wgengine/netstack/netstack.go:395]: Where is destination ip?

Comment (bradfitz) [reply wgengine/netstack/netstack.go:395]: Whoops, I hadn't tested this on Windows yet 😛 

Fixed.

**Comment by bradfitz** [comment]
I verified this works on Windows.

**Comment by bradfitz** [comment]
Whoops, I realized I forgot to test IPv6 and one ping to ipv6.google.com and ....

```
authReconfig: ra=false dns=true 0x01: <nil>
panic: IP4Header called on non-IPv4 Parsed

goroutine 39440 [running]:
tailscale.com/net/packet.(*Parsed).ICMP4Header(0xc0004ac7b0)
        /home/bradfitz/src/tailscale.com/net/packet/packet.go:343 +0x17b
tailscale.com/wgengine/netstack.(*Impl).injectInbound(0xc000238540, 0xc000764b00, 0xc0004e9bc0)
        /home/bradfitz/src/tailscale.com/wgengine/netstack/netstack.go:411 +0x33e
tailscale.com/net/tstun.(*Wrapper).filterIn(0xc0001d0000, {0xc002800010, 0x38, 0xffef})
        /home/bradfitz/src/tailscale.com/net/tstun/wrap.go:601 +0x78d
tailscale.com/net/tstun.(*Wrapper).Write(0xc0001d0000, {0xc002800000, 0xc000505680, 0x1}, 0x10)
        /home/bradfitz/src/tailscale.com/net/tstun/wrap.go:614 +0x7c
golang.zx2c4.com/wireguard/device.(*Peer).RoutineSequentialReceiver(0xc0007f2e00)
        /home/bradfitz/go/pkg/mod/golang.zx2c4.com/wireguard@v0.0.0-20211116201604-de7c702ace45/device/receive.go:477 +0x4d1
created by golang.zx2c4.com/wireguard/device.(*Peer).Start
        /home/bradfitz/go/pkg/mod/golang.zx2c4.com/wireguard@v0.0.0-20211116201604-de7c702ace45/device/peer.go:199 +0x295
```

**Comment by bradfitz** [comment]
PTAL, now with working IPv6 pings

**Review by crawshaw — APPROVED** [review]

**Comment by crawshaw** [comment]
It's interesting to think about what would need to be adapted in the ubuntu vm integration test to test this. Potentially not that much.

**Pull request merged**